react/features/mobile/navigation/components/settings/components/SettingsNavigationContainer.js

```diff
@@ -0,0 +1,82 @@
+import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import HelpView from '../../../../../settings/components/native/HelpView';
+import PrivacyView from '../../../../../settings/components/native/PrivacyView';
+import SettingsView
+    from '../../../../../settings/components/native/SettingsView';
+import TermsView from '../../../../../settings/components/native/TermsView';
+import { screen } from '../../../routes';
+import {
+    linkScreenOptions,
+    navigationContainerTheme,
+    settingsScreenOptions,
+    welcomeScreenOptions
+} from '../../../screenOptions';
+import {
+    settingsNavigationContainerRef
+} from '../SettingsNavigationContainerRef';
+
+const SettingsStack = createStackNavigator();
+
+
+/**
+ * The type of the React {@code Component} props of {@link SettingsNavigationContainer}.
+ */
+type Props = {
+
+    /**
+     * Callback to be invoked when settings screen is focused.
+     */
+    onSettingsScreenFocused: Function
+};
+
+
+const SettingsNavigationContainer = ({ onSettingsScreenFocused }: Props) => {
```

Comment:
Please rename `onSettingsScreenFocused` to `isInWelcomePage` (and make it a boolean) and we're good!

Reply:
Done.